deploy/helm/sumologic/templates/_helpers/_metrics.tpl

```diff
@@ -36,10 +36,10 @@ Generate list of remoteWrite endpoints for telegraf configuration
 {{- $endpoints := list -}}
 {{- $kps := get .Values "kube-prometheus-stack" -}}
 {{- range $remoteWrite := $kps.prometheus.prometheusSpec.remoteWrite }}
-{{- $endpoints = append $endpoints ($remoteWrite.url | trimPrefix "http://$(METADATA_METRICS_SVC).$(NAMESPACE).svc.cluster.local.:9888" | quote) -}}
+{{- $endpoints = append $endpoints ($remoteWrite.url | trimPrefix "http://$(METADATA_METRICS_SVC).$(NAMESPACE):9888" | quote) -}}
 {{- end }}
 {{- range $remoteWrite := $kps.prometheus.prometheusSpec.additionalRemoteWrite }}
-{{- $endpoints = append $endpoints ($remoteWrite.url | trimPrefix "http://$(METADATA_METRICS_SVC).$(NAMESPACE).svc.cluster.local.:9888" | quote) -}}
+{{- $endpoints = append $endpoints ($remoteWrite.url | trimPrefix "http://$(METADATA_METRICS_SVC).$(NAMESPACE):9888" | quote) -}}
 {{- end -}}
```

Comment:
Why in configs you add `.Values.sumologic.clusterDNSDomain` but here you simply remove the suffix?

Reply:
I had to remove it in the remoteWrite configuration in values.yaml, so I thought I'd be consistent about it.